lib/git-shell-out-strategy.js

```diff
@@ -19,6 +19,7 @@ import GitTimingsView from './views/git-timings-view';
 import WorkerManager from './worker-manager';
 
 const LINE_ENDING_REGEX = /\r?\n/;
+const MAX_STATUS_OUTPUT_LENGTH = 1024 * 1024 * 10;
```

Comment:
Any idea what this roughly translates to in terms of file count?

We wouldn't want to accidentally lock out .com or Linux kernel devs 😅

Reply:
Unfortunately it depends on path lengths. So deeply nested projects will be penalized more.